crates/kitsune-test/src/lib.rs

```diff
@@ -11,8 +11,23 @@ use std::{env, error::Error, panic};
 
 mod catch_panic;
 
+#[doc(hidden)]
+pub use http;
+#[doc(hidden)]
+pub use hyper;
+
 type BoxError = Box<dyn Error + Send + Sync>;
 
+#[macro_export]
+macro_rules! build_ap_response {
+    ($body:expr) => {
+        $crate::http::Response::builder()
+            .header("Content-Type", "application/activity+json")
+            .body($crate::hyper::Body::from($body))
+            .unwrap()
+    };
+}
```

Comment:
This might be just a matter of taste, but I'd avoid creating a macro unless necessary. The macro would lead to confusing errors when `Body` isn't in the caller's scope, for example.

```suggestion
use http::{header::CONTENT_TYPE, Response};
use hyper::Body;
// …
pub fn build_ap_response<B>(body: B) -> Response<Body>
where
    B: From<Body>,
{
    Response::builder()
        .header(CONTENT_TYPE, "application/activity+json")
        .body(Body::from(body))
        .unwrap()
}
```

Reply:
Replaced this with a function